README.md

```diff
@@ -56,6 +56,80 @@ Since this library is used by Home Assistant, see <https://www.home-assistant.io
 
 So follow that advice and try to scrape all fields at once, similar to the `get_form_action_url_and_hidden_inputs` in helpers.py.
 
+## Example Usage - CLI
+
+The opower library comes with a CLI interface to get familiar with the library and read actual data from your utility.
+
+```sh
+# Show information on how to call the library CLI interface
+python -m opower --help
+
+# Example: show recent usage only, for all accounts:
+python -m opower --utility comed --username <username> --password <password> --usage_only --aggregate_type day
+[...]
+start_time	end_time	consumption	start_minus_prev_end	end_minus_prev_end
+2025-06-10 00:00:00-05:00	2025-06-11 00:00:00-05:00	-9.44	None	None
+2025-06-11 00:00:00-05:00	2025-06-12 00:00:00-05:00	-3.825	0:00:00	1 day, 0:00:00
+2025-06-12 00:00:00-05:00	2025-06-13 00:00:00-05:00	9.6225	0:00:00	1 day, 0:00:00
+2025-06-13 00:00:00-05:00	2025-06-14 00:00:00-05:00	13.6575	0:00:00	1 day, 0:00:00
+2025-06-14 00:00:00-05:00	2025-06-15 00:00:00-05:00	-0.665	0:00:00	1 day, 0:00:00
+2025-06-15 00:00:00-05:00	2025-06-16 00:00:00-05:00	-4.5125	0:00:00	1 day, 0:00:00
+
+```
+
+## Example Usage - in Python
```

Comment:
I rather not have to maintain the code in the README and in `__main__.py`. Whoever is interested in using the library in Python should be able to find `__main__.py`. But if you think it's valuable, can you link to `__main__.py` instead? Full link like the pge.py link above because the README is also used in the pypi package.